Extension/src/githubAPI.ts

```diff
@@ -130,7 +130,17 @@ export interface BuildInfo {
  */
 export async function getTargetBuildInfo(updateChannel: string): Promise<BuildInfo> {
     return getReleaseJson()
-        .then(builds => getTargetBuild(builds, updateChannel))
+        .then(builds => {
+            // If the user version is greater than or incomparable to the latest available verion then there is no need to update
+            // Allows testing pre-releases without accidentally downgrading to the latest version
+            const userVersion: PackageVersion = new PackageVersion(util.packageJson.version);
+            const latestVersion: PackageVersion = new PackageVersion(builds[0].name);
+            if (userVersion.isGreaterThan(latestVersion) || (userVersion.suffix && userVersion.suffix !== 'insiders')) {
```

Comment:
might want to do a starts with 'insiders' or contains 'insiders' since we have the insiders2 problem right now.

Reply:
for insiders2, the 2 will be removed and `.suffix` will just have `insiders` and `.suffixVersion` will have 2